packages/pivotql-parser-expression/generate.js

```diff
@@ -29,7 +29,7 @@ const grammar = {
       ['true|false', 'return "BOOLEAN";'], // true/false
       ['null|undefined', 'return "PRIMITIVE";'],
       [
-        '[A-zÀ-ÖØ-öø-įĴ-őŔ-žǍ-ǰǴ-ǵǸ-țȞ-ȟȤ-ȳɃɆ-ɏḀ-ẞƀ-ƓƗ-ƚƝ-ơƤ-ƥƫ-ưƲ-ƶẠ-ỿ_][\\.A-zÀ-ÖØ-öø-įĴ-őŔ-žǍ-ǰǴ-ǵǸ-țȞ-ȟȤ-ȳɃɆ-ɏḀ-ẞƀ-ƓƗ-ƚƝ-ơƤ-ƥƫ-ưƲ-ƶẠ-ỿ_-]*',
+        '[A-zÀ-ÖØ-öø-įĴ-őŔ-žǍ-ǰǴ-ǵǸ-țȞ-ȟȤ-ȳɃɆ-ɏḀ-ẞƀ-ƓƗ-ƚƝ-ơƤ-ƥƫ-ưƲ-ƶẠ-ỿ_0-9][\\.A-zÀ-ÖØ-öø-įĴ-őŔ-žǍ-ǰǴ-ǵǸ-țȞ-ȟȤ-ȳɃɆ-ɏḀ-ẞƀ-ƓƗ-ƚƝ-ơƤ-ƥƫ-ưƲ-ƶẠ-ỿ_0-9-]*',
```

Comment:
I'm curious, why did you allow to start a Symbol with a number? Other languages usually forbid it. There are some interesting pros and cons [here](https://softwareengineering.stackexchange.com/questions/133381/why-should-identifiers-not-begin-with-a-number). What do you think?

Reply:
I will take a look, thank you. I agree with you, it is a bad idea to allow a Symbol to start with a  number but I work with data that is created by users that are not aware of good practice. This case has already happened.